docker/alba_centos7_jenkins/Dockerfile

```diff
@@ -126,6 +126,14 @@ RUN git clone https://github.com/domsj/orocksdb.git \
 # be uniform on shared lib locations
 RUN cp /usr/local/lib/librocksdb.so* /lib64/
 
+#for now, instll ordma manually
+RUN yum -y install librdmacm-devel
+RUN git clone https://github.com/toolslive/ordma.git \
+    && cd ordma \
+    && git checkout tags/0.0.2 \
```

Comment:
update to current latest commit hash on ordma master as you might want this change too https://github.com/toolslive/ordma/commit/7ef1886d11fd7dd50fc1e307278e874006906ad7 ?
(same remark for other docker files & travis script)